build.sbt

```diff
@@ -0,0 +1,5 @@
+name := "smart-words"
+
+version := "0.1"
+
+scalaVersion := "3.1.1"
```

Comment:
Scala set to version 3 but when running then output is 2.
According to www it's because I'm not using Scala3 specific features. Otherwise IntelliJ plugin will implicitly do the update.